### PR TITLE
Add example for correlation matrix.

### PR DESCRIPTION
**Pull request by AndreasAlbert**

### CODE REVIEW

**Comment by clelange** [comment]
That is a very good example to have!
Direct link for testing: https://mybinder.org/v2/gh/HEPData/hepdata_lib/example_correlation?filepath=examples/correlation.ipynb

I will have a look and come back with comments if I have any.

File: examples/correlation.ipynb [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    "The input histogram looks like this:\n",
+    "<img src=\"correlation.png\" width=\"450\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reading the histogram into python\n",
+    "Now that we have our example input, let's read it back. Reading from ROOT files into python is performed by the RootFileReader class."
```
Comment (clelange) [inline examples/correlation.ipynb]: `RootFileReader`

File: examples/correlation.ipynb [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+   ],
+   "source": [
+    "print(data[\"x_edges\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In any case, you do not have to worry too much about the format, because we will directly feed these numbers into the Variable objects for our HEPData entry, just as we would do in the other examples."
```
Comment (clelange) [inline examples/correlation.ipynb]: `Variable`

**Comment by AndreasAlbert** [comment]
New changes:
* Committed the plot to the repository so that it shows up without having to actually run the notebook.
* Adopted the `style` according to @clelange's suggestion
* Replaced the calls to `print` 
* Fixed a typo in `dependent`

**Pull request merged**